src/plugins/lowMach.hpp

```diff
@@ -5,6 +5,6 @@ namespace lowMach
 {
 void setup(nrs_t* nrs, dfloat gamma0);
 void buildKernel(occa::properties kernelInfo);
-void qThermalIdealGasSingleComponent(dfloat time, occa::memory o_div);
+void qThermalIdealGasSingleComponent(dfloat time, occa::memory o_div,occa::memory o_alphav);
```

Comment:
Please don't change this function. Instead add a new `qThermalRealGasSingleComponent`